TMS/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path , include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('',include('transfers.urls')),
```

Comment:
instead of `''`, put `'TMS'`
Also, for the built-in `auth.views` we need
`path('TMS', include('django.contrib.auth.views')`

in a nutshell, this file would look like:


`path('TMS-admin', include('django.contrib.auth.urls')`
`path('TMS', include('django.contrib.auth.views')`
`path('TMS', include('transfers.urls')`